webapp/src/lib/components/spot-listings/detail-modal.svelte

```diff
@@ -7,6 +7,8 @@
 
     export let listing : ParkingSpot;
 
+    const defaultZoom : number = 13;
```

Comment:
we have a file for constant btw lib/constant.ts. put it there if you want, though you don't have to do it for this PR